Userland/Libraries/LibCompress/PackBitsDecoder.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "PackBitsDecoder.h"
+
+namespace Compress::PackBits {
+
+ErrorOr<ByteBuffer> decode_all(ReadonlyBytes bytes, Optional<u64> expected_output_size)
+{
+    auto memory_stream = make<FixedMemoryStream>(bytes);
+
+    ByteBuffer decoded_bytes;
+    TRY(decoded_bytes.try_ensure_capacity(expected_output_size.value_or(0)));
```

Comment:
```suggestion
    if (expected_output_size)
        TRY(decoded_bytes.try_ensure_capacity(expected_output_size.value());
```

Just for clarity's sake, I don't expect this to have an impact on performance.